warehouse/templates/pages/help.html

```diff
@@ -25,6 +25,7 @@
 {% macro publishing() %}How do I package and publish my code for PyPI?{% endmacro %}
 {% macro trove_classifier() %}What's a trove classifier?{% endmacro %}
 {% macro verified_email() %}Why do I need a verified email address?{% endmacro %}
+{% macro compromised_password() %}Why won't PyPI accept my password?{% endmacro %}
```

Comment:
Sometimes PyPI doesn't accept a password because it's inaccurate (e.g., the caps lock key is on), so I suggest rewording this to include a key phrase from the error message "This password has appeared in a breach or has otherwise been compromised and cannot be used."

Reply:
"Why is PyPI telling me my password is compromised?"